fbw-a380x/src/systems/instruments/src/ND/NDSimvarPublisher.tsx

```diff
@@ -34,11 +33,10 @@ export enum NDVars {
   ilsCourse = 'L:A32NX_FM_LS_COURSE',
   selectedHeading = 'L:A32NX_FCU_HEADING_SELECTED',
   showSelectedHeading = 'L:A320_FCU_SHOW_SELECTED_HEADING',
-  pposLat = 'PLANE LATITUDE', // TODO replace with fm position
-  pposLong = 'PLANE LONGITUDE', // TODO replace with fm position
   absoluteTime = 'E:ABSOLUTE TIME',
   kccuOnL = 'L:A32NX_KCCU_L_KBD_ON_OFF',
   kccuOnR = 'L:A32NX_KCCU_R_KBD_ON_OFF',
+  baroMode = 'KOHLSMAN SETTING STD',
```

Comment:
The PFD uses Simplane for this via `SimplaneValueProvider`, which calls Simplane, and that ultimately reads from the LVAR `L:XMLVAR_Baro1_Mode` and does the following.
```js
if (val == 0)
                return "QFE";
            else if (val == 1)
                return "QNH";
            else
                return "STD";
```
We should probably not invent something new here that might get unexpectedly broken and won't show up when we search to rip all this out (already mostly done on the A320 in the FMGC PR).

Reply:
Ah, I thought SimplaneValueProvider was some old pattern we should avoid. I moved it to MsfsAvCommon, and also use it for the ND now.

Reply:
It's preferable to use the same old pattern rather than introducing another pattern that's not right :D.